### ISSUE TITLE
Discrepancy: `params` Defined as Array of Integer in Apipie appears as Array of Strings in Swagger File

### ISSUE THREAD

**Issue by rahulramm** [Apipie/apipie-rails#903]
Defining an array parameter as `Integer` within Apipie validates correctly in the HTML documentation but generates the Swagger file with an array of `string` type.

**Parameter defined within Apipie**
```
param :folder_ids, Array, of: Integer, desc: "Array of folder IDs"
```

**Expected Behavior:**
The Swagger file should reflect the parameter folder_ids as an array of `Integer` type, consistent with the definition in Apipie's DSL.

**Actual Behavior:**
The Swagger file represents folder_ids as an array of `string` type, contrary to the defined type in Apipie.

Screenshots:

**1. HTML Documentation (Apipie):**

<img width="644" alt="Screenshot 2023-12-17 at 9 08 25 PM" src="https://github.com/Apipie/apipie-rails/assets/108332798/3523c1c9-14bc-429d-b97f-d7fecc76e47d">

**2. Swagger File Output:**

<img width="494" alt="Screenshot 2023-12-17 at 9 07 59 PM" src="https://github.com/Apipie/apipie-rails/assets/108332798/78d66cfe-5e01-4236-8119-a9fbfea1defa">

``` 
{
    "name": "folder_ids",
    "type": "array",
    "items": {
      "type": "string"
    },
    "in": "formData",
    "description": "Array of folder IDs"
}
```

**Comment by mathieujobin** [comment]
I see, thanks for reporting.

@PanosCodes is it possible for you to look at this one? Would you know already where this `string` type be coming from?

Thanks

**Comment by PanosCodes** [comment]
Feel free to test https://github.com/Apipie/apipie-rails/pull/904

**Comment by mathieujobin** [comment]
Thanks @rahulramm for the report
Thanks @PanosCodes for the quick fix.

released in v1.3.0

**Comment by rahulramm** [comment]
Much Appreciated. Thank you so much for the quick turnaround. @mathieujobin @PanosCodes
